script/update.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -v            # print commands before execution, but don't expand env vars in output
+set -o errexit    # always exit on error
+set -o pipefail   # honor exit codes when piping
+set -o nounset    # fail on unset variables
+
+git clone "https://electron-bot:$GH_TOKEN@github.com/electron/electron-api-historian" module
+cd module
+git submodule update --init
+npm ci
+
+git config user.email electron@github.com
+git config user.name electron-bot
+
+# Update the electron submodule
+pushd electron && git checkout origin/master && popd
+# bail if the submodule sha didn't change
+if [ "$(git status --porcelain -- electron)" = "" ]; then
+  echo "electron origin/master ref has not changed; goodbye!"
+  exit 78
```

Comment:
why not zero?

Reply:
78 is the GitHub Actions "neutral" exit code, which doesn't cause a failure but also doesn't cause a green check mark. I chose it because it will allow folks to see at a glance when a job is stopping early because it's not detecting any changes.